scripts/run_development_container

```diff
@@ -1,9 +1,51 @@
 #!/bin/bash
-sudo docker run -it \
+set -euo pipefail
+
+usage()
+{
+    echo "Usage: $0 IP [OPTION]"
+
+    echo "IP is the ip address of the host machine (dev server). Only evaluated when --dev-server flag is passed."
```

Comment:
Is this is being run on the dev server, can't it figure out it's own IP like you have in the other script?

Reply:
Yes it is being run on the dev server when the --dev-server option is passed. Although now the IP is found in the script not passed in.